src/jit/codegenxarch.cpp

```diff
@@ -8380,6 +8380,11 @@ void CodeGen::genCreateAndStoreGCInfoX64(unsigned codeSize, unsigned prologSize
         // frame
         gcInfoEncoder->SetSizeOfEditAndContinuePreservedArea(preservedAreaSize);
     }
+    if (compiler->opts.IsReversePInvoke())
+    {
+        assert(compiler->lvaReversePInvokeFrameVar != BAD_VAR_NUM);
+        gcInfoEncoder->SetReversePInvokeFrameSlot(compiler->lvaReversePInvokeFrameVar);
```

Comment:
Something like `compiler->lvaTable[compiler->lvaReversePInvokeFrameVar].lvStkOffs` should be sufficient to retrieve the offset of the variable on the frame. IIRC, however, that will be the offset from the frame pointer (whether it is in rbp or rsp). @BruceForstall can you confirm?